crates/ty_python_semantic/src/types.rs

```diff
@@ -1094,7 +1097,8 @@ impl<'db> Type<'db> {
             | Type::ModuleLiteral(_)
             | Type::ClassLiteral(_)
             | Type::SpecialForm(_)
-            | Type::IntLiteral(_) => self,
+            | Type::IntLiteral(_)
+            | Type::TypedDict(_) => self,
```

Comment:
We'll need to do more here in due course, I think (but fine to leave this like this for now, of course!)